staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -327,22 +322,41 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaWorkloadLeaderElection = newFlowSchema(
-		"workload-leader-election", "leader-election", 200,
+	// We add an explicit rule for endpoints-controller with high precedence
+	// to ensure that those calls won't get catch by the following
+	// <workload-leader-election> flow-schema.
+	//
+	// TODO(#80289): Get rid of this rule once we get rid of support for
+	//   using endpoints and configmaps objects for leader election.
+	SuggestedFlowSchemaEndpointsController = newFlowSchema(
+		"endpoints-controller", "workload-high", 150,
 		flowcontrol.FlowDistinguisherMethodByUserType,
 		flowcontrol.PolicyRulesWithSubjects{
-			Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
+			Subjects: append(
+				users(user.KubeControllerManager),
+				kubeSystemServiceAccount("endpoints-controller")...),
 			ResourceRules: []flowcontrol.ResourcePolicyRule{
 				resourceRule(
 					[]string{"get", "create", "update"},
 					[]string{corev1.GroupName},
 					[]string{"endpoints", "configmaps"},
```

Comment:
no need to include configmaps here, right?

Reply:
@liggitt : why do we not need configmaps here?  Leader elections can still be done using configmaps, right?

Reply:
this isn't the leader-election flow, it's the endpoints controller one

Reply:
Ah, right.  Thanks!

Reply:
@liggitt I fixed it in #107054

Reply:
Didn't hurt, but true - we can remove it. Done.